tests/cnf/core/network/sriov/tests/qinq.go

```diff
@@ -885,3 +1065,133 @@ func runQinQDpdkTestCases(nodeName, serverName, clientName, sriovNetworkName, na
 	By("Validate that the TCP traffic is double tagged")
 	readAndValidateTCPDump(tcpDumpPod, []string{"bash", "-c", "tail -20 /tmp/tcpdump"}, outPutSubString)
 }
+
+func defineNetworkBondAnnotation(sVlan, cvlan string, ipAddresses []string) []*multus.NetworkSelectionElement {
+	bondInterface := pod.StaticIPBondAnnotationWithInterface("nadcvlan100", "bond0.100",
+		[]string{sVlan, sVlan, cvlan}, ipAddresses)
+
+	return bondInterface
+}
+
+// defineBondNAD returns network attachment definition for a Bond interface.
+func defineQinQBondNAD(nadname, mode string) *nad.Builder {
+	bondNad, err := nad.NewMasterBondPlugin(nadname, mode).WithFailOverMac(1).
+		WithLinksInContainer(true).WithMiimon(100).WithVLANInContainer(uint16(100)).
```

Comment:
remove .WithMiimon(100)

Reply:
removed

Reply:
hmm. it's weird. could be a bug

Reply:
Investigating

Reply:
**Not sure why it is waiting for a miion value?**
noreUnknown=1;K8S_POD_NAMESPACE=sriov-tests;K8S_POD_NAME=server-1ad;K8S_POD_INFRA_CONTAINER_ID=9f64183755417ed32573509b036b5c713498bb9141446516e5a206dadc086998;K8S_POD_UID=ea211c55-1a6e-47f4-b68f-093ba24dfb16" Path:"" ERRORED: error configuring pod [sriov-tests/server-1ad] networking: [sriov-tests/server-1ad/ea211c55-1a6e-47f4-b68f-093ba24dfb16:nadmasterbond0]: error adding container to network "nadmasterbond0": Failed to create bonded link (net3), error: **Failed to convert bondMiimon value () to an int, error: strconv.Atoi: parsing "": invalid syntax**
': StdinData: {"binDir":"/var/lib/cni/bin","clusterNetwork":"/host/run/multus/cni/net.d/10-ovn-kubernetes.conf","cniVersion":"0.3.1","daemonSocketDir":"/run/multus/socket","globalNamespaces":"default,openshift-multus,openshift-sriov-network-operator","logLevel":"verbose","logToStderr":true,"name":"multus-cni-network","namespaceIsolation":true,"type":"multus-shim"}